packages/calypso-e2e/src/lib/components/page-template-modal-component.ts

```diff
@@ -16,17 +16,17 @@ const selectors = {
  */
 export class PageTemplateModalComponent {
```

Comment:
We probably need to refactor this modal component at some point in the future to fit with the new refactor, and be included on the editor page. That's out of scope for this PR. So for now, I just clean up the typings/names to reflect that this could be both a `Page` or `Frame` (for atomic support).

Reply:
Good call out, I had totally forgot about this component during the refactor. I can work on this next once you merge this in?